Makefile

```diff
@@ -33,26 +33,6 @@ queryservice-ui:
 	bash update_cache.sh services
 	eval ". ./build/build_queryservice_ui.sh; bash build/build_queryservice_ui_docker.sh ${QUERYSERVICE_UI_IMAGE_NAME}"
 
-oauth: 
```

Comment:
Are these removed on purpose?

Reply:
they are built in the dockerfile for the bundle now

Reply:
Ah, makes sense… in that case the definition of `all` should be changed to no longer include these.